### PR TITLE
[Sarah] Accept this Mr. Cholo.

### PR DESCRIPTION
**Pull request by SamuelChanMD**
Some content changes as well as minor formatting

### CODE REVIEW

File: public/css/home.css [hunk 1]
```diff
@@ -606,7 +611,7 @@ h2 {
     background-color: #000;
 }
 
-.registryOptionText {
+.registry-option-text {
```
Comment (ploh007) [inline public/css/home.css:614]: why was this class name changed?

File: resources/views/home.blade.php [hunk 2]
```diff
@@ -367,7 +367,7 @@
 </div>
 <div class='jumbotron wow slideInUp' id='photos-container' data-wow-offset='300'>
     <div class='container text-center'>
-        <h1 class='heading'>Engagement Photos</h1>
+        <h1 class='heading'>Just Some Photos</h1>
```
Comment (ploh007) [inline resources/views/home.blade.php:370]: Lol

File: resources/views/our_story.blade.php [hunk 3]
```diff
@@ -1,17 +1,17 @@
-<div class='col-md-12'>
+<div class='col-md-8 col-md-offset-2'>
```
Comment (ploh007) [inline resources/views/our_story.blade.php:1]: My bad here. Linting and refactoring led to undesired formatting changes to this.

File: public/css/home.css (hunk 1 above)
Comment (SamuelChanMD) [reply public/css/home.css:614]: Cause on the home blade, it is written like that. I'm guessing it got updated with lint, to keep standardization of class names?

File: resources/views/our_story.blade.php (hunk 3 above)
Comment (SamuelChanMD) [reply resources/views/our_story.blade.php:1]: No worries, you're still a good friend

**Review by ploh007 — APPROVED** [review]

**Pull request merged**